ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -182,72 +202,125 @@ const POPULATIONS = [
     queryParams: { [GENOME_VERSION_37]: 'dataset=gnomad_sv_r2_1' },
     helpMessage: GNOMAD_SV_CRITERIA_MESSAGE,
   },
+]
+
+const MITO_POPULATIONS = [
+  {
+    field: 'callset',
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
```

Comment:
As ever, copy paste is bad. You should define a `CALLSET_POP` constant and use it in the main `POPULATIONS` and then extend it here: `{ ...CALLSET_POP, titleContainer: sectionTitle, section: HOM_SECTION }`